File: pulsar/java/src/main/java/com/examples/hello/pulsar/Application.java
package com.examples.hello.pulsar;

import io.micronaut.runtime.Micronaut;

public class Application {
  public static void main(String[] args) {
    Micronaut.build(args)
      .mainClass(Application.class)
      .defaultEnvironments("dev")
      .banner(true)
      .start();
  }
}

File: pulsar/kotlin/src/main/kotlin/com/example/hello/pulsar/Application.kt
package com.example.hello.pulsar

import io.micronaut.runtime.Micronaut
import io.swagger.v3.oas.annotations.OpenAPIDefinition
import io.swagger.v3.oas.annotations.info.Info


@OpenAPIDefinition(
  info = Info(
    title = "hello.pulsar",
    version = "0.1.0",
    description = "Kotlin Example Project"
  )
)
//@OpenAPIManagement
//@OpenAPIInclude(classes = [ServiceController::class])
object Application {
  @JvmStatic
  fun main(args:Array<String>) {
    Micronaut.run(Application.javaClass, *args)
  }
}
